splunk_topology/check.py

```diff
@@ -131,6 +132,9 @@ def _dispatch_and_await_search(self, instance, saved_searches):
 
         for (sid, saved_search) in search_ids:
             self.log.debug("Processing saved search: %s." % saved_search.name)
+            if sid is None:
+                self.log.warn("Skipping the saved search %s as it doesn't exist " % saved_search.name)
```

Comment:
Is it the only case that when `sid is None` that the saved search does not exists?

Reply:
yes, when the saved search doesn't exist, `sid will be None` as post API call ignores the error and returns None.